source/MaterialXRenderGlsl/GlslValidator.h

```diff
@@ -39,7 +40,7 @@ class GlslValidator : public ShaderValidator
 {
   public:
     /// Create a GLSL validator instance
-    static GlslValidatorPtr create();
+    static GlslValidatorPtr create(unsigned int dim = 1024);
```

Comment:
Not sure why the default has changed from 512 to 1024? This will slow down unit tests.